lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -594,8 +594,10 @@ def _create_node(
 
         # Create libvirt domain (i.e. VM).
         xml = self._create_node_domain_xml(environment, log, node)
+        log.debug(f"Domain xml for {node_context.vm_name} - {xml}")
```

Comment:
This log line seems unnecessary. If the user ever needs to view this, they can just use `virsh dumpxml`.

Reply:
Right. But that is only when the Guest VM is running. It doesn't help much in cases when VM's are cleanup. But I felt this would be of no harm and is good to have.

Reply:
Are you aware of the `keep-environment` flag?

Reply:
Yes I'm aware, but we cannot use it in our autmated pipelines because of resource constraints.